app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_comments_count_on_create
+
+  def update_comments_count_on_create
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ OPTIONAL ] Hey there! 🌟 Your code is fantastic! To take it up a notch, I'd suggest marking the method responsible for updating the comment counter as private. 🛡️ Why? Because you've already set up a Rails active callback that fires when a new comment is added. This callback then calls the method to update the comment counter. 🔄 By marking it private, you're signaling that it should only be accessed within the class, keeping things neat and secure. 👌 Great job! 🚀